src/main/webapp/cache-details/cache-details.html

```diff
@@ -83,63 +83,51 @@
                         <th>Name</th>
                         <th>Average reads</th>
                         <th>Average writes</th>
+                        <th>Average replication time</th>
                         <th>Total entries</th>
                         <th>Hits</th>
                         <th>Misses</th>
-                        <th>Read/Write ratio</th>
-                        <th>Total failed writes</th>
                     </thead>
                     <tr  ng-repeat="stats in currentCacheStats.nodeStats">
                         <td>{{stats['node-name']}}</td>
                         <td>{{stats['average-read-time']}} ms</td>
                         <td>{{stats['average-write-time']}} ms</td>
+                        <td>{{stats['average-replication-time']}}</td>
                         <td>{{stats['number-of-entries']}}</td>
                         <td>{{stats['hits']}}</td>
                         <td>{{stats['misses']}}</td>
-                        <td>{{stats['read-write-ratio']}}</td>
-                        <td>N/A</td>
                     </tr>
                 </table>
             </div>
         </div>
 
         <br>
-        <div class="row">
+      <div class="row">
             <div class="col-md-12">
                 <div class="cache-status">
-                    <span class="cache-status-label">Properties</span>
+                    <span class="cache-status-label">Configuration</span>
                 </div>
             </div>
+      </div>
+      <div class="cache-properties row">
+        <div><p><p>
         </div>
-
-        <div class="cache-properties row">
-            <div class="col-md-3">
-                <div>Cache name:</div>
-                <div>Mode:</div>
-                <div>Associated cache/loader store:</div>
-
-                <div>Write-through/Write-behind:</div>
-                <div>Isolation level:</div>
-                <div>Eviction strategy:</div>
-
-                <div>Passivation:</div>
-                <div>Expiration configuration:</div>
-                <div>Cross-site replication:</div>
-            </div>
-            <div class="col-md-9">
-                <div>{{currentCache.name}}</div>
-                <div>{{currentCache.type}}</div>
-                <div>N/A</div>
-
-                <div>Mock</div>
-                <div>{{currentCache.locking.LOCKING.isolation}}</div>
-                <div>N/A</div>
-
-                <div>N/A</div>
-                <div>N/A</div>
-                <div>N/A</div>
-            </div>
+        <div ng-if="currentCache.isDistributed()">
```

Comment:
This ng-if part appears on multiple places through app, so we could eliminate it by moving this if-part to the cache-configuration directive. End result: we would just call `<cache-configuration cache-type="distributed-cache" ...></cache-configuration>`, and directive would inside itself take care that correct html is displayed. I will comment more about that in the directive code.